internal/app/router.go

```diff
@@ -3,14 +3,21 @@ package app
 import (
 	"github.com/AlexanderVasiliev23/yp-url-shortener/internal/app/handlers/add"
 	"github.com/AlexanderVasiliev23/yp-url-shortener/internal/app/handlers/get"
+	"github.com/AlexanderVasiliev23/yp-url-shortener/internal/app/handlers/shorten"
+	"github.com/AlexanderVasiliev23/yp-url-shortener/internal/app/middlewares/gzip"
+	"github.com/AlexanderVasiliev23/yp-url-shortener/internal/app/middlewares/logger"
 	"github.com/labstack/echo/v4"
 )
 
-func (a *app) configureRouter() *echo.Echo {
+func (a *App) configureRouter() *echo.Echo {
 	e := echo.New()
 
-	e.POST("/", add.Add(a.localStorage, a.tokenGenerator, a.conf.BaseAddress))
-	e.GET("/:token", get.Get(a.localStorage))
+	e.Use(logger.Middleware(a.logger))
```

Comment:
```suggestion
	e.Use(logger.Middleware(a.logger), gzip.Middleware())
```
их вроде можно через запятую перечислить

Reply:
Для наглядности хотел бы так оставить. Особенно будет актуально, когда будет больше мидлвар.

Reply:
их можно в каждой строке отдельно писать - тут вопрос форматирования, зато все мидлвары в одном скоупе и не потеряются